reactive/kubernetes_worker.py

```diff
@@ -789,18 +789,29 @@ def configure_kubelet(dns, ingress_ip):
                     'clientCAFile': str(ca_crt_path)
                 }
             },
+            'authorization': {
+                'mode': 'Webhook'
+            },
             'clusterDomain': dns['domain'],
             'failSwapOn': False,
             'port': 10250,
+            'protectKernelDefaults': True,
+            'readOnlyPort': 0,
             'tlsCertFile': str(server_crt_path),
             'tlsPrivateKeyFile': str(server_key_path)
         }
         if dns['enable-kube-dns']:
             kubelet_config['clusterDNS'] = [dns['sdn-ip']]
+
+        # Handle feature gates
+        feature_gates = {}
+        if get_version('kubelet') >= (1, 19):
+            # NB: required for CIS compliance
+            feature_gates['RotateKubeletServerCertificate'] = True
```

Comment:
The comment from https://github.com/charmed-kubernetes/charm-kubernetes-master/pull/104/files#diff-bf8eaea470605940d2c1bc8d1a7ecd39R2221 applies here too.  I don't *think* this really affects us given our 10 yr certs and the fact that we don't use kube csr.  I conditioned it on 1.19+ just to be safe in case previous versions had shorter cert expirations.